### PR TITLE
Feature/31 add app description

### PR DESCRIPTION
**Pull request by NkwaTambe**

### CODE REVIEW

File: docs/power-payment-app.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+D --> K
+E --> K
+J --> L
+K -->|Route| A
+
+
+```
+
+
+###Deployment and Operation:
```
Comment (stephane-segning) [inline docs/power-payment-app.md:76]: Separate this title from the `#`

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work. Please fix this typo error

File: docs/power-payment-app.md (hunk 1 above)
Comment (NkwaTambe) [reply docs/power-payment-app.md:76]: ok sir

Comment (NkwaTambe) [reply docs/power-payment-app.md]: Hello sir @stephane-segning  please review the new changes.

**Review by stephane-segning — APPROVED** [review]
Good work!

**Pull request merged**